DESCRIPTION

```diff
@@ -28,13 +28,15 @@ Imports:
     lifecycle,
     openssl (>= 2.0.0),
     packrat (>= 0.6),
+    renv,
```

Comment:
Not sure if you need it for this PR, but if you'd like to depend on `renv` internals without risking breakage from new `renv` releases, you can also use `renv::embed()` to embed a local copy of `renv` here.

Reply:
Related thought: Do we need a version constraint on renv?

Reply:
We don't need to embed because we're only using exported functions so far, but will keep it in mind for the future. 

I've add a requirement for the latest version from CRAN.